client/cypress/integration/visualizations/choropleth_spec.js

```diff
@@ -36,23 +36,19 @@ describe("Choropleth", () => {
       cy.visit(`queries/${id}/source`);
       cy.getByTestId("ExecuteButton").click();
     });
+    cy.getByTestId("NewVisualization").click();
+    cy.getByTestId("VisualizationType").selectAntdOption("VisualizationType.CHOROPLETH");
   });
 
   it("creates visualization", () => {
-    cy.clickThrough(`
-      NewVisualization
-      VisualizationType
-      VisualizationType.CHOROPLETH
-    `);
-
     cy.clickThrough(`
       VisualizationEditor.Tabs.General
       Choropleth.Editor.MapType
       Choropleth.Editor.MapType.countries
       Choropleth.Editor.KeyColumn
       Choropleth.Editor.KeyColumn.name
-      Choropleth.Editor.TargetField
-      Choropleth.Editor.TargetField.name
+      Choropleth.Editor.KeyType
+      Choropleth.Editor.KeyType.name
```

Comment:
```suggestion
      Choropleth.Editor.TargetField
      Choropleth.Editor.TargetField.name
```

Yep, same thing, looks like you are running the web server without #5186, `npm run watch` should solve it locally :)